docs/fundamentals/toc.yml

```diff
@@ -934,22 +934,24 @@ items:
             href: ../standard/commandline/index.md
           - name: Get started tutorial
             href: ../standard/commandline/get-started-tutorial.md
-          - name: Command-line syntax
+          - name: Command-line syntax overview
             href: ../standard/commandline/syntax.md
-          - name: Define commands
-            href: ../standard/commandline/define-commands.md
-          - name: Model binding
-            href: ../standard/commandline/model-binding.md
-          - name: Tab completion
-            href: ../standard/commandline/tab-completion.md
-          - name: Dependency injection
-            href: ../standard/commandline/dependency-injection.md
-          - name: Customize help
-            href: ../standard/commandline/customize-help.md
-          - name: Handle termination
-            href: ../standard/commandline/handle-termination.md
-          - name: Use middleware
-            href: ../standard/commandline/use-middleware.md
+          - name: How to
+            items:
+            - name: Parse and invoke the result
+              href: ../standard/commandline/parse-and-invoke.md
+            - name: Customize parsing and validation
+              href: ../standard/commandline/parsing-and-validation.md
+            - name: Configure the parser
+              href: ../standard/commandline/command-line-configuration.md
+            - name: Enable and customize tab completion
+              href: ../standard/commandline/tab-completion.md
+            - name: Customize help
+              href: ../standard/commandline/help.md
+          - name: Design guidance
```

Comment:
I'm not sure why this is down at the bottom, would it make more sense to have it above the how to area?

Reply:
I would say it's more advanced content for people who build more complex apps.